Moya.podspec

```diff
@@ -1,6 +1,6 @@
 Pod::Spec.new do |s|
   s.name         = "Moya"
-  s.version      = "11.0.2"
+  s.version      = "12.0.0-beta.1"
```

Comment:
I think this should be `12.0.0` or at least `12.0.0-beta.2`.

Reply:
This will be done by the release script after merging into `master`